wasmedge-ggml-llama-interactive/README.md

```diff
@@ -58,3 +91,7 @@ What if I have 3 apples?
 Answer:
 The total cost of 3 apples would be 15 dollars. Each apple costs 5 dollars, so 3 apples would cost 3 x 5 = 15 dollars.
 ```
+
+## Credit
+
+The metal file, `ggml-metal.metal` is from [llama.cpp](git://github.com/ggerganov/llama.cpp.git@b1217).
```

Comment:
Let's just acknowledge llama.cpp in general. The plugin itself was built in their work, not just the metal file.

Reply:
Fixed.